atc/integration/active_users_test.go

```diff
@@ -0,0 +1,28 @@
+package integration_test
+
+import (
+	"time"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/atc/atccmd"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+)
+
+var _ = Describe("ATC Integration Test", func() {
+	It("active users are tracked", func() {
+		client := login(atcURL, "test", "test")
+		twoMonthsAgo := time.Now().AddDate(0, -2, 0)
+		Eventually(func() []atc.User {
+			users, _ := client.ListActiveUsersSince(twoMonthsAgo)
+			return users
+		}, atccmd.BatcherInterval).Should(
```

Comment:
Since this is a global now, can we modify it to be shorter to make the test go by faster?

Reply:
That said, setting it to `1*time.Second` doesn't really seem to make the test much faster anyway - it typically takes ~3s for me with the default `atccmd.BatcherInterval` which isn't too bad

Reply:
yeah i love the idea of tuning it down, but i think that's a problem for another day -- in general i want to be especially careful and intentional making changes to this sort of thing because Billing.